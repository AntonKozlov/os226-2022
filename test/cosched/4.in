# 	entryid	cnt	prio	deadline
coapp	1	4	2	0
coapp	2	16	0	0

cosched 1
