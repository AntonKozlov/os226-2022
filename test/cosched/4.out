      coapp_task id 1 cnt 4
        coapp_rt id 2 cnt 16
        coapp_rt id 2 cnt 15
      coapp_task id 1 cnt 3
        coapp_rt id 2 cnt 14
        coapp_rt id 2 cnt 13
      coapp_task id 1 cnt 2
        coapp_rt id 2 cnt 12
        coapp_rt id 2 cnt 11
      coapp_task id 1 cnt 1
        coapp_rt id 2 cnt 10
        coapp_rt id 2 cnt 9
      coapp_task id 1 cnt 0
        coapp_rt id 2 cnt 8
        coapp_rt id 2 cnt 7
        coapp_rt id 2 cnt 6
        coapp_rt id 2 cnt 5
        coapp_rt id 2 cnt 4
        coapp_rt id 2 cnt 3
        coapp_rt id 2 cnt 2
        coapp_rt id 2 cnt 1
        coapp_rt id 2 cnt 0
